tools/copy_bom/src/util.rs

```diff
@@ -10,6 +10,26 @@ use std::path::PathBuf;
 use std::process::{Command, Output};
 use std::vec;
 
+lazy_static! {
+    /// This map stores the path of occlum-modified loaders.
+    /// The `key` is the name of the loader. The `value` is the loader path.
+    static ref OCCLUM_LOADERS: HashMap<&'static str, &'static str> = {
+        let mut m = HashMap::new();
+        // glibc version
+        m.insert("ld-linux-x86-64.so.2", "/opt/occlum/glibc/lib/ld-linux-x86-64.so.2");
```

Comment:
Maybe we could maintain the map in a config file to handle a more complex situation. For example, user install occlum musl in another path.

Reply:
fixed. Now add a new config file to read loaders.